application/analyst/services/tension_analyzer.py

```
@@ -166,19 +169,25 @@ def _parse_response(self, response: str) -> TensionDiagnosis:
-                suggestions=data["suggestions"]
+                diagnosis=str(data.get("diagnosis", "暂无诊断信息")),
+                tension_level=str(data.get("tension_level", "low")),
+                missing_elements=[str(x) for x in data.get("missing_elements", [])],
+                suggestions=[str(x) for x in data.get("suggestions", [])]
             )
-        except (json.JSONDecodeError, KeyError) as e:
-            # 如果解析失败，返回默认结果
+        except KeyError as e:
             return TensionDiagnosis(
-                diagnosis=f"解析响应失败: {str(e)}",
+                diagnosis=f"解析响应失败: 缺少字段 {str(e)}",
```

Ruff (0.15.10)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)